config/crd/bases/autoscaling.katalyst.kubewharf.io_katalystverticalpodautoscalers.yaml

```diff
@@ -568,45 +568,54 @@ spec:
                   constraints.
                 properties:
                   algorithmPolicy:
-                    description: policy of algorithm, if no algorithm is provided,
-                      using default
+                    description: policy of algorithm, if not provided, using default
                     properties:
                       algorithm:
-                        description: Algorithm is the recommended algorithm of choice
+                        default: percentile
+                        description: Algorithm specify the recommendation algorithm
```

Comment:
Probably better to change the description to: algorithm specifies the recommendation algorithm used by the default recommender, default to "percentile"